docs/faq/questions/cloud-faq.mdx

```diff
@@ -55,6 +55,22 @@ Cloud.
 A [user](/guides/cloud/users) is anyone with a login to Cypress Cloud who has
 been invited to see and review the test results of your organization.
 
+## <Icon name="angle-right" /> What if I can't access my Cypress Cloud account?
+
+1. If you haven't signed up, see the
+   [Sign Up with an Invitation](/guides/cloud/users#Sign-Up-with-an-Invitation)
+   instructions
+
+2. Try resetting your password. If you never received the email, try the
+   following:
+
+- Try signing in with Google or GitHub
```

Comment:
Let's tab these bullets in one space. Makes it a but easier to read with the hierarchy.